src/Business Foundation/App/NoSeries/src/Upgrade/NoSeriesInstaller.Codeunit.al

```diff
@@ -34,4 +35,27 @@ codeunit 329 "No. Series Installer"
 
         UpgradeTag.SetUpgradeTag(NoSeriesUpgradeTags.GetImplementationUpgradeTag());
     end;
+
+    local procedure TriggerMovedTableSchemaCheck()
+    var
+        NoSeries: Record "No. Series";
+        NoSeriesLine: Record "No. Series Line";
+        NoSeriesRelationship: Record "No. Series Relationship";
+        NoSeriesTenant: Record "No. Series Tenant";
+#if not CLEAN24
+        NoSeriesLineSales: Record "No. Series Line Sales";
+        NoSeriesLinePurchase: Record "No. Series Line Purchase";
+#endif
+    begin
+#pragma warning disable AA0175
+        if NoSeries.FindFirst() then;
```

Comment:
Why do we need this code? First iteration was throwing an error and this iteration is simply reading a row?
Why don't we provide a proper error message except the error of not finding first?
Should we emit telemetry if we cannot find it?

Reply:
it was requested by server as an extra check to load schema metadata just to make sure... the if is there in case the table is empty, which is fine. There's nothing to be done if the code passes. If it fails then it would have failed to sync schema metadata and code execution would stop anyways...

Reply:
Server runtime requested the change as an extra sanity check. The if .. then is there so code doesn't fail when there is no data (the data itself is not relevant). There is no need for an error or telemetry. If the code runs all is good, if it fails it would fail due to a schema bug on the server which would stop all .al code execution anyway.